Error while checking build/O2/fullCI for fa90e54a5ffd17267dae71767d0b9c5f6b7dd8ca at 2022-11-18 21:30:
```
## sw/BUILD/O2-latest/log
/sw/SOURCES/O2/10318-slc8_x86-64/0/Detectors/CTP/reconstruction/include/CTPReconstruction/CTFHelper.h:41:35: error: 'const struct o2::ctp::LumiInfo' has no member named 'lumiCountsFV0'; did you mean 'countsFV0'?
/sw/SOURCES/O2/10318-slc8_x86-64/0/Detectors/CTP/reconstruction/include/CTPReconstruction/CTFHelper.h:41:35: error: 'const struct o2::ctp::LumiInfo' has no member named 'lumiCountsFV0'; did you mean 'countsFV0'?
/sw/SOURCES/O2/10318-slc8_x86-64/0/Detectors/CTP/reconstruction/include/CTPReconstruction/CTFHelper.h:41:35: error: 'const struct o2::ctp::LumiInfo' has no member named 'lumiCountsFV0'; did you mean 'countsFV0'?
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/10318/fa90e54a5ffd17267dae71767d0b9c5f6b7dd8ca/build_O2_fullCI/pretty.html).

--- a/DataFormats/Detectors/CTP/include/DataFormatsCTP/CTF.h
+++ b/DataFormats/Detectors/CTP/include/DataFormatsCTP/CTF.h
@@ -27,14 +27,15 @@ namespace ctp
 
 /// Header for a single CTF
 struct CTFHeader : public o2::ctf::CTFDictHeader {
-  uint64_t lumiCounts = 0; /// Luminosity counts moving average over lumiNHBFs orbits
+  uint64_t lumiCounts = 0;    /// FT0 Luminosity counts moving average over lumiNHBFs orbits
+  uint64_t lumiCountsFV0 = 0; /// FV0 Luminosity counts moving average over lumiNHBFs orbits
   uint32_t lumiNHBFs = 0;  /// Number of HBFs over which lumi is integrated
   uint32_t lumiOrbit = 0;  /// 1st orbit of TF where lumi was updated, can be compared with firstOrbit
   uint32_t nTriggers = 0;  /// number of triggers
   uint32_t firstOrbit = 0; /// orbit of 1st trigger
   uint16_t firstBC = 0;    /// bc of 1st trigger
 
-  ClassDefNV(CTFHeader, 2);
+  ClassDefNV(CTFHeader, 3);
 };
 
 /// wrapper for the Entropy-encoded trigger inputs and classes of the TF

--- a/DataFormats/Detectors/CTP/include/DataFormatsCTP/LumiInfo.h
+++ b/DataFormats/Detectors/CTP/include/DataFormatsCTP/LumiInfo.h
@@ -24,10 +24,13 @@ struct LumiInfo {
   LumiInfo() = default;
   uint32_t orbit = 0;       // orbit of TF when was updated
   uint32_t nHBFCounted = 0; // length of interval in HB
-  uint64_t counts = 0;      // counts in the interval
+  uint64_t counts = 0;      // counts in the interval for the nominal lumi detector (FT0)
+  uint64_t countsFV0 = 0;   // connts for FV0 (less reliable)
   float getLumi() const { return nHBFCounted > 0 ? float(counts / (nHBFCounted * o2::constants::lhc::LHCOrbitMUS * 1e-6)) : 0.f; }
+  float getLumiFV0() const { return nHBFCounted > 0 ? float(countsFV0 / (nHBFCounted * o2::constants::lhc::LHCOrbitMUS * 1e-6)) : 0.f; }
   float getLumiError() const { return nHBFCounted > 0 ? float(std::sqrt(counts) / (nHBFCounted * o2::constants::lhc::LHCOrbitMUS * 1e-6)) : 0.f; }
-  ClassDefNV(LumiInfo, 1);
+  float getLumiFV0Error() const { return nHBFCounted > 0 ? float(std::sqrt(countsFV0) / (nHBFCounted * o2::constants::lhc::LHCOrbitMUS * 1e-6)) : 0.f; }
+  ClassDefNV(LumiInfo, 2);
 };
 } // namespace ctp
 

--- a/Detectors/CTP/reconstruction/include/CTPReconstruction/CTFHelper.h
+++ b/Detectors/CTP/reconstruction/include/CTPReconstruction/CTFHelper.h
@@ -38,7 +38,7 @@ class CTFHelper
   CTFHeader createHeader(const LumiInfo& lumi)
   {
     CTFHeader h{o2::detectors::DetID::CTP, 0, 1, 0, // dummy timestamp, version 1.0
-                lumi.counts, lumi.nHBFCounted, lumi.orbit,
+                lumi.counts, lumi.countsFV0, lumi.nHBFCounted, lumi.orbit,
                 uint32_t(mData.size()), 0, 0};
     if (mData.size()) {
       h.firstOrbit = mData[0].intRecord.orbit;

